modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+
+* With this update, repository URLs are now validated during creation to ensure they are properly formatted and use valid schemes, such as `http` or `https`. This enhancement helps prevent misconfigurations and runtime errors.
+
+* With this update, `PipelineRun` status comments now render correctly in markdown on Bitbucket Data Center and Bitbucket Cloud, instead of appearing as raw strings in the pull request UI.
+
+* With this update, when the `generateSigningSecret` key is enabled, the {pipelines-title} Operator automatically creates Cosign key pairs, specifically `cosign.key` (private key) and `cosign.pub` (public key). These key pairs are then stored within the `signing-secrets` secret, located in the `tekton-pipelines` namespace. You can enable the `generateSigningSecret` key by setting it to `true` in the `TektonConfig` custom resource.
++
+.Example of enabling Cosign key pairs
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    generateSigningSecret: true
+----
+
+* With this update, users can retrieve logs from Splunk that were forwarded by OpenShift Logging. To configure this feature, you can set environment variables such as `SPLUNK\_SEARCH\_TOKEN, LOGGING\_PLUGIN\_QUERY\_PARAMS` (including index details such as index=konflux), and `LOGGING\_PLUGIN\_API\_URL` (with the Splunk endpoint and port number).
+
+* With this update, the {tekton-results} watcher leverages `StatefulSet` ordinals for improved high availability and workload distribution as an alternative to leader election.
++
+--
+.Example enabling the StatefulSet ordinals for the {tekton-results} watcher
+[source,yaml]
+----
+spec:
+  chains:
+    performance:
+      disable-ha: false
+      buckets: 4
+      replicas: 4
+      statefulset-ordinals: true
+----
+
+:FeatureName: Using `StatefulSet` ordinals for high availability
+include::snippets/technology-preview.adoc[]
+--
+
+* With this update, the {tekton-results} API supports response filtering to minimize payload size and enhance network efficiency. This update enables users to define the specific fields included in API responses, which significantly benefits List operations by preventing the retrieval of entire objects, thus optimizing performance.
+
+* With this update, tasks in the web console now feature `displayName` properties, providing clearer, more user-friendly names within the {pipelines-title} interface. This behavior replaces the use of technical task names and brings the `displayName` parameter functionality, previously found in TektonDashboard, to the web console.
+
+* With this update, {pac} extends GitOps workflow support in GitLab by enabling GitOps comments on pushed commits. This enhancement allows users to receive pipeline status updates directly on commit events, in addition to merge requests. Previously, this feature was only available for GitHub. # (Need to confirm if this is added in Google Docs)
```

Comment:
> `# (Need to confirm if this is added in Google Docs)`

I'm not an asciidoc person so want to confirm this is not left in by mistake 😛

Reply:
Fixed.